src/dragonball/src/dbs_virtio_devices/src/lib.rs

```diff
@@ -201,15 +205,20 @@ pub enum Error {
     #[error("virtio-vsock error: {0}")]
     VirtioVsockError(#[from] self::vsock::VsockError),
 
-    #[cfg(feature = "virtio-net")]
-    #[error("Virtio-net error: {0}")]
-    VirtioNetError(#[from] crate::net::NetError),
-
     #[cfg(feature = "virtio-fs")]
     /// Error from Virtio fs.
     #[error("virtio-fs error: {0}")]
     VirtioFs(fs::Error),
 
+    #[cfg(feature = "virtio-net")]
+    #[error("virtio-net error: {0}")]
+    VirtioNet(#[from] NetError),
```

Comment:
should we change transparency error transform to explicit error transform

Reply:
same below

Reply:
Fixed.